vk/backend/backend.go

```diff
@@ -101,7 +102,16 @@ func RunWithBackend(ctx context.Context, runner *runner.Runner, statuses *os.Fil
 	}
 
 	requests := pod.Spec.Containers[0].Resources.Requests
-	disk := requests["titus/disk"]
+
+	var disk resource.Quantity
+
+	for _, k := range []v1.ResourceName{"titus/disk", v1.ResourceEphemeralStorage, v1.ResourceStorage} {
```

Comment:
I think if multiple labels are assigned we should error out, right?

Reply:
My guess is they will set both during migration. We simply pick the first one we encounter.

Reply:
I believe they will not set both. We will advertise both, but they will only pick one at any given time.  Regardless, still looks fine.